modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -83,15 +81,18 @@ In your network, you might need to whitelist the hostnames and ports of external
 [[install_Istio]]
 == Download, Install, and Configure Istio
 
-You can install Istio 1.6.x (`1.6.0` to `1.6.8`) or 1.7.x in your environment to support Anypoint Service Mesh.
+Anypoint Service Mesh supports Istio versions 1.6.0 through 1.6.8 and 1.7.0. You can install any of these Istio versions in your environment.
 
 === Prerequisites
 
 Before you begin, ensure that you download Istio using the https://istio.io/docs/setup/[Istio Documentation].
 
 === Install and Configure Istio
 
-To install and configure Istio for Anypoint Service Mesh, run the following command from your CLI:`$ istioctl install`.
+To install and configure Istio for Anypoint Service Mesh, run the following command from your terminal:
```

Comment:
"Istio for Anypoint Service Mesh" sounds like the name of one product.
I think you mean this: "Istio for use with Anypoint Service Mesh".

Reply:
Done! Thanks!